pkg/image/validation/os_test.go

```diff
@@ -566,3 +571,73 @@ func TestValidateUnattended(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateRawConfiguration(t *testing.T) {
+	tests := map[string]struct {
+		Definition             image.Definition
+		ExpectedFailedMessages []string
+	}{
+		`not included`: {
+			Definition: image.Definition{},
+		},
+		`diskSize specified and valid`: {
+			Definition: image.Definition{
+				Image: image.Image{
+					ImageType: image.TypeRAW,
+				},
+				OperatingSystem: image.OperatingSystem{
+					RawConfiguration: image.RawConfiguration{
+						DiskSize: "64G",
+					},
+				},
+			},
+		},
+		`diskSize invalid as invalid suffix`: {
+			Definition: image.Definition{
+				Image: image.Image{
+					ImageType: image.TypeRAW,
+				},
+				OperatingSystem: image.OperatingSystem{
+					RawConfiguration: image.RawConfiguration{
+						DiskSize: "130B",
+					},
+				},
+			},
+			ExpectedFailedMessages: []string{
+				fmt.Sprintf("the 'rawConfiguration/diskSize' field must require an integer followed by an 'M/G/T' suffix to indicate size when 'imageType' is '%s'.", image.TypeRAW),
+			},
+		},
+		`diskSize invalid as zero`: {
+			Definition: image.Definition{
+				Image: image.Image{
+					ImageType: image.TypeRAW,
+				},
+				OperatingSystem: image.OperatingSystem{
+					RawConfiguration: image.RawConfiguration{
+						DiskSize: "0G",
+					},
+				},
+			},
```

Comment:
Can you add tests for:
* Negative number
* No number at all but does end with the letter M, G, or T
* Lower case for M, G, or T (no need to do all three, just one)